packages/orbit-components/src/Box/Box.stories.js

```diff
@@ -0,0 +1,272 @@
+    const justify = select("justify", JUSTIFY, "start");
+    const wrap = select("wrap", WRAP, WRAP[0]);
+
+    return (
+      <Box
+        display={display}
+        justify={justify}
+        background="blueNormal"
+        height="full"
+        wrap={wrap}
+        align={align}
+        direction={direction}
+      >
+        <div style={{ height: 50, width: 50, background: "#ccc" }} />
+        <div style={{ height: 50, width: 50, background: "#ccc" }} />
+      </Box>
+    );
+  })
+  .add(
+    "Playground",
+    () => {
+      const children = text("Children", "Box");
+      const display = select("display", DISPLAY, DISPLAY.FLEX);
+      const overflow = select("overflow", OVERFLOW, OVERFLOW.VISIBLE);
+      const wrap = select("wrap", WRAP, WRAP[0]);
+      const shrink = select("shrink", SHRINK, SHRINK[0]);
+      const grow = select("grow", GROW, GROW[0]);
+      const align = select("align", ALIGN, ALIGN.START);
+      const textAlign = select("text-align", TEXT_ALIGN, TEXT_ALIGN.LEFT);
+      const justify = select("justify", JUSTIFY, JUSTIFY.CENTER);
+      const direction = select("direction", DIRECTION, DIRECTION.ROW);
+      const width = select("width", WIDTH_AND_HEIGHT, WIDTH_AND_HEIGHT.AUTO);
+      const maxWidth = text("max-width", "100px");
+      const height = select("height", WIDTH_AND_HEIGHT, WIDTH_AND_HEIGHT.AUTO);
+      const maxHeight = text("max-height", "100px");
+      const elevation = select("elevation", ELEVATION, ELEVATION.ACTION);
+      const borderRadius = select("border-radius", BORDER_RADIUS, BORDER_RADIUS.NORMAL);
+      const position = select("position", POSITION, POSITION.RELATIVE);
+      const top = text("top", "10px");
+      const right = text("right", "10px");
+      const bottom = text("left", "10px");
+      const left = text("bottom", "10px");
+      const margin = select("margin", SPACINGS, SPACINGS[0]);
+      const padding = object("padding", { top: "small", left: "x-small", right: "xx-small" }, "");
+
+      return (
+        <Box
+          display={display}
+          wrap={wrap}
+          shrink={shrink}
+          grow={grow}
+          align={align}
+          textAlign={textAlign}
+          direction={direction}
+          justify={justify}
+          width={width}
+          maxWidth={maxWidth}
+          height={height}
+          maxHeight={maxHeight}
+          elevation={elevation}
+          borderRadius={borderRadius}
+          position={position}
+          top={top}
+          right={right}
+          bottom={bottom}
+          left={left}
+          overflow={overflow}
+          padding={padding}
+          margin={margin}
+          color="productLight"
+          background="inkLight"
+        >
+          {children}
+        </Box>
+      );
+    },
+    {
+      info: "Some description about this type of component. ",
+    },
+  )
+  .add(
+    "RTL",
+    () => (
+      <RenderInRtl>
+        <Box>Box in RTL</Box>
```

Comment:
Having screenshotted this won't be efficient enough.

We should add properties that need to be RTL aware, e.g. `left`, `right`, horizontal `margin` and `padding` and `textAlign`.